plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2337,8 +2337,21 @@ elementgui.block.properties_redstone=Redstone properties
 elementgui.block.redstone_connect=<html>Does redstone connect to this block?<br><small>Power works even if this is unchecked
 elementgui.block.emits_redstone=Does emit redstone?
 elementgui.block.redstone_power=Emitted redstone power:
+elementgui.block.custom_properties_states=Custom block properties
+elementgui.block.custom_properties.add=Add custom property
+elementgui.block.custom_properties.add.title=New block state property
+elementgui.block.custom_properties.add.input=Property name
+elementgui.block.custom_properties.add.error_duplicate=Property with this name is already defined.
+elementgui.block.custom_properties.add.error_duplicate.title=Duplicate property
```

Comment:
Where is this key used? Given I have left comments on this before, I would have almost assumed you have checked this yourself. Also the key above?

Reply:
I remember checking all these keys and seeing them being used at least once, except those 3 you noted.

Reply:
Afaik it is 2

Reply:
I meant the keys you commented on earlier.

Reply:
Ok but where are those two then used? I have asked this but did not get the answer yet

Reply:
These are indeed not used since the validation is performed directly inside text field.